niworkflows/viz/utils.py

```diff
@@ -65,19 +66,24 @@ def save_html(template, report_file_name, unique_string, **kwargs):
 
 
 def as_svg(image, filename='temp.svg'):
-    ''' takes an image as created by nilearn.plotting and returns a blob svg.
-    A bit hacky. '''
+    ''' Takes an image as created by nilearn.plotting and returns the svg with
+        extra headers removed and the width and height style values set to
+        nothing. '''
     image.savefig(filename)
     with open(filename, 'r' if PY3 else 'rb') as file_obj:
```

Comment:
We should replace writing svgs to the filesystem, in favor of something like https://github.com/oesteban/mriqc/blob/master/mriqc/viz/svg.py

Maybe for another PR?